local/config/finna/WayfinderService.ini.sample

```diff
@@ -0,0 +1,3 @@
+[General]
+url = "http://localhost"
```

Comment:
I mean all libraries. Since a Finna instance can display items from several libraries, there must be library-specific configuration unless everything is shared by all libraries.

Reply:
There's still really no relationship between a Finna view with its configuration, and a specific library. The holdings information of any library could be displayed by any Finna view, and particularly finna.fi that brings all libraries etc. together displays records and holdings from all libraries. This means that you cannot tie Wayfinder configuration to a Finna view. It must be specific to a library system identified by the source identifier that's also a prefix in each record ID. So there needs to be configuration that can map a source ID to a Wayfinder URL and adapter.

Reply:
Yes. In Callnumber helper's callnumber method it's the first parameter, `$source`.